app/src/main/java/com/google/droidjump/AchievementsFragment.java

```diff
@@ -65,12 +70,17 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         View rootView = getView();
         RecyclerView achievementsView = rootView.findViewById(achievementsViewId);
         LoadingHelper.onLoading(activity, rootView, achievementsViewId);
-        activity.getAchievementsClient().load(true).addOnCompleteListener(activity, task -> {
+        activity.getAchievementsClient().load(false).addOnCompleteListener(activity, task -> {
```

Comment:
can you add a comment with what is false/true mean in this case?
 
it may be forceReload but not sure.
So it should look like /* forceReload= */